### PR TITLE
Enclose paths with quotes in testPackages.proj

### PR DESCRIPTION
**Pull request by carlossanlop**
Enclosing these two paths with double quotes fixes the following error:

```
D:\runtime> .\build.cmd-subset libs.tests -runtimeConfiguration release -allconfigurations

 Determining projects to restore...
Tool 'coverlet.console' (version '1.7.0') was restored. Available commands: coverlet
Tool 'dotnet-reportgenerator-globaltool' (version '4.5.2') was restored. Available commands: reportgenerator
Restore was successful.
All projects are up-to-date for restore.
Determining projects to restore...
All projects are up-to-date for restore.
*** Restoring ***
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
D:\runtime\src\libraries\pkg\test\testPackages.proj(190,5): error MSB3073: The command "C:\Program Files\dotnet\dotnet restore --packages "D:\runtime\artifacts\bin\testPackages" /p:LocalPackagesPath=D:\runtime
\artifacts\packages\Debug\Shipping\ /nr:false /warnaserror "D:\runtime\artifacts\bin\testPkg/support/test.msbuild"" exited with code 9009.
Build FAILED.
D:\runtime\src\libraries\pkg\test\testPackages.proj(190,5): error MSB3073: The command "C:\Program Files\dotnet\dotnet restore --packages "D:\runtime\artifacts\bin\testPackages" /p:LocalPackagesPath=D:\runtime
\artifacts\packages\Debug\Shipping\ /nr:false /warnaserror "D:\runtime\artifacts\bin\testPkg/support/test.msbuild"" exited with code 9009.
 0 Warning(s)
 1 Error(s)
Time Elapsed 00:00:03.98
Build failed.
Some builds failed:
Configuration: Debug, Architecture: x64
```

### CODE REVIEW

**Review by safern — APPROVED** [review]

File: src/libraries/pkg/test/testPackages.proj [hunk 1]
```diff
@@ -177,7 +177,7 @@
           Condition="'$(ArchiveTests)' != 'true'">
 
     <PropertyGroup>
-      <TestRestoreCommand>$(TestDotNetPath)</TestRestoreCommand>
+      <TestRestoreCommand>&quot;$(TestDotNetPath)&quot;</TestRestoreCommand>
```
Comment (ViktorHofer) [inline src/libraries/pkg/test/testPackages.proj:180]: as we are inside a propertygroup you can just use the `"` here.

Comment (carlossanlop) [reply src/libraries/pkg/test/testPackages.proj:180]: Ok sure, thanks for confirming that. I wasn't sure the character was allowed.

Comment (carlossanlop) [reply src/libraries/pkg/test/testPackages.proj:180]:
```suggestion
      <TestRestoreCommand>"$(TestDotNetPath)"</TestRestoreCommand>
```

File: src/libraries/pkg/test/testPackages.proj [hunk 2]
```diff
@@ -195,7 +195,7 @@
           Condition="'$(ArchiveTests)' != 'true'">
 
     <PropertyGroup>
-      <TestBuildCommand>$(TestDotNetPath)</TestBuildCommand>
+      <TestBuildCommand>&quot;$(TestDotNetPath)&quot;</TestBuildCommand>
```
Comment (carlossanlop) [inline src/libraries/pkg/test/testPackages.proj:198]:
```suggestion
      <TestBuildCommand>"$(TestDotNetPath)"</TestBuildCommand>
```

**Review by ViktorHofer — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Comment by carlossanlop** [comment]
@safern @ViktorHofer the CI failed, but I'm not sure if I broke it or it's an unrelated timeout. Should I rerun it?

**Comment by ViktorHofer** [comment]
We can safely merge the change as it only affects the allconfigurations leg which succeeded. Thanks.

**Pull request merged**